src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
now it will only work with this selector, it should all work for all of them
<img width="542" alt="Знімок екрана 2023-11-01 о 15 25 18" src="https://github.com/mate-academy/layout_product-cards/assets/106310796/23d71327-d1c9-4053-b2c9-c7a55d988df6">